ci-operator/config/openshift/aws-account-operator/openshift-aws-account-operator-master.yaml

```diff
@@ -42,10 +42,32 @@ tests:
   commands: make validate
   container:
     from: src
-- as: test-int
-  commands: echo "Test Integration WIP"
-  container:
-    from: src
+- as: integration-test
```

Comment:
is there a way to make this step to not prevent merging if it fails? that would be convenient while we develop this.

Reply:
According to this link, assigning `optional` to `true` will not prevent merging if the CI step fails:- https://docs.ci.openshift.org/docs/architecture/ci-operator/#pre-submit-tests

I will add `optional: true` to this PR.